AWSIoTTests/AWSIoTDataManagerTests.swift

```diff
@@ -741,18 +741,35 @@ class AWSIoTDataManagerTests: XCTestCase {
         let testMessage = "Test Message"
         let testTopic = "TestTopic"
         gotMessage.expectedFulfillmentCount = 5
+
+        let subAckExpectation = self.expectation(description: "Subscription should be acknowledged")
+        let subAckCallback: AWSIoTMQTTAckBlock = {
```

Comment:
Might be better to just call this ackCallback since ackCallback is already a term?

Reply:
Changed.